packages/data-connect/package.json

```diff
@@ -31,10 +31,11 @@
     "prettier": "prettier --write '*.js' '*.ts' '@(src|test)/**/*.ts'",
     "build:deps": "lerna run --scope @firebase/'{app,data-connect}' --include-dependencies build",
     "dev": "rollup -c -w",
-    "test": "run-p --npm-path npm test:emulator",
+    "test": "run-p --npm-path npm lint test:emulator",
```

Comment:
I'm guessing what you're trying to do here is when you run tests locally, you use the emulator, but when they're run in CI, you don't. But I think you should probably also run node tests in CI? The default test command run by ci is "test:ci" which you already have in this file and won't be used for anything else. I think what you want to do is change these 3 lines:

```
    "test": "run-p --npm-path npm lint test:emulator",
    "test:ci": "node ../../scripts/run_tests_in_ci.js -s test:all",
    "test:all": "run-p --npm-path npm lint test:node test:browser",
```
in this file and that's it. And don't need to make any changes in run_tests_in_ci.js.